.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
Could you explain the purpose of this?

Reply:
https://stackoverflow.com/a/75245402/23000795